### ISSUE TITLE
GPG verification

### ISSUE THREAD

**Issue by indutny** [gypkg/gypkg#1]
For security-related repos like https://github.com/gypkg/openssl it may be beneficial to have some sort of GPG verification for a git tags.

Here is what I had in mind:

```
git://github.com/org/repo@semver => main.gyp:target [flags]
```

Where flags is a comma-separated list of values. For GPG it could be:

```
[gpg:pool-name]
```

Where `pool-name` will result in `~/.gypkg/gpg/pool-name`, and will contain ids of keys that are authorized to sign the package. If `gypkg gen` will encounter new key - it will ask user to confirm addition of it to the `~/.gypkg/gpg/pool-name` file.

Does it make sense? Does anyone have better idea?

**Comment by indutny** [comment]
I think it makes sense to require https://github.com/cgwalters/git-evtag, since otherwise git tags could be collision-unsafe in near future (SHA-1)

**Comment by indutny** [comment]
Guess it should be not a pool, but rather a gpg key ring file name, so we could use http://superuser.com/questions/399938/how-to-create-additional-gpg-keyring

**Comment by indutny** [comment]
Created https://github.com/indutny/git-secure-tag . It has verification API, so it may be used, I guess?

**Comment by indutny** [comment]
Something landed: f5736e2
